package.json

```diff
@@ -97,6 +97,7 @@
     "event-target-shim": "^5.0.1",
     "fbjs-scripts": "^1.1.0",
     "hermes-engine": "~0.6.0",
+    "hermes-engine-darwin": "~0.6.0",
```

Comment:
This should be a peer dependency, because auto-linking will otherwise always try to enable it when it's installed, no?

Reply:
Well, I will double check that tomorrow but...theoretically, it could be a peer dependency, but for a different reason.

Auto linking will only link first-level dependencies, so the ones you have defined in your `package.json` yourself, not the ones inside React Native.

Making it a `peerDependency` would make sense if this package was meant to be installed by end users *in order to enable Hermes*. 

Since we plan to a) turn this to be the default in the future, b) make it turnable by a change to a flag, having it as a part of dependencies is a natural direction.

Reply:
> Since we plan to make it turnable by a change to a flag, having it as a part of dependencies makes it a bit easier.

I see what you mean, but I don’t really like the idea of pulling in large artefacts for platforms one may not use. However, I guess the same applies to the android engine one, which is already listed as a plain dependency.

Long term I want to get rid of those flags in the Podfile and the need for a separate file that specifies all transitive dependencies, but that will take a while longer.

In the end, I’m on the fence about doing this now. Do others have opinions?

Reply:
> Long term I want to get rid of those flags in the Podfile and the need for a separate file that specifies all transitive dependencies, but that will take a while longer.

I like that goal. Would love to learn more on how this could be achieved, but I guess that's something we can discuss in parallel!